config/manager/kustomization.yml

```diff
@@ -1,2 +1,8 @@
 resources:
-- manager.yaml
\ No newline at end of file
+- manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: ghcr.io/spinkube/spin-operator
+  newName: ghcr.io/spinkube/spin-operator
+  newTag: dev
```

Comment:
I don't believe we want this getting checked in.

Reply:
I think this question is still relevant

Reply:
slipped. It's removed now